apps/studio/src/constants/formBuilder.ts

```diff
@@ -1,6 +1,6 @@
 // Rankings are adapted from JSONForms' Material Renderers package
 export const JSON_FORMS_RANKING = {
-  ArrayControl: 3,
+  ArrayControl: 4,
```

Comment:
actually how do we select the number here? is there a logic to it

Reply:
I just copied over the ListWithDetail tester that I based this sub-component off from: https://github.com/eclipsesource/jsonforms/blob/0fd6329a0f08ae2f74e312f0dbb2a727abbddf77/packages/material-renderers/src/additional/MaterialListWithDetailRenderer.tsx#L177-L180